### DISCUSSION TITLE
tailind css

### DISCUSSION THREAD

**Discussion by madfortech** [misterdebug/crud-generator-laravel#6] [General]
Before I install this crud, support tailwind classes ?

**Comment by misterdebug** [comment]
Hi @madfortech, nope sorry and not planned (i don't use tailwind) but you can customize very easily :)

**Reply by misterdebug** [reply]
https://github.com/misterdebug/crud-generator-laravel/wiki/Custom-your-views

**Reply by misterdebug** [reply]
Hi @madfortech, tailwind will be added in the next few weeks

**Reply by madfortech** [reply]
thanks
